networking/ciao-cnci-agent/scripts/generate_cnci_cloud_image.sh

```diff
@@ -64,18 +64,9 @@ fi
 echo -e "\nMounting image: $image"
 tmpdir=$(mktemp -d)
 
-#it can take some time for the device to get created
-retry=0
-loop=''
-until [ $retry -ge 3 ]
-do
-    if [ "$loop" == "" ]; then
-	loop=`sudo losetup -f --show -P $image`
-    fi
-    sudo mount ${loop}p$partition "$tmpdir" && break
-    let retry=retry+1
-    echo "Mount failed, retrying $retry"
-done
+loop=`sudo losetup -f --show -P $image`
+sudo udevadm settle
+sudo mount ${loop}p$partition "$tmpdir" && break
 
 if [ $retry -ge 3 ]
```

Comment:
we probably need a better way now to check if the mount actually went thro. Maybe look for some file that needs to be present.